client/src/partials/users/user.modal.html

```diff
@@ -0,0 +1,103 @@
+<form name="UserForm" bh-submit="UserModalCtrl.submit(UserForm)" bh-form-defaults novalidate>
+  <div class="modal-header">
+  <ol class="headercrumb">
+    <li ng-if="UserModalCtrl.isCreating" class="title">
+      {{ "USERS.ADDING_USER" | translate }}
+      <label class="label label-warning">{{ "USERS.CREATE" | translate }}</label>
+    </li>
+    <li ng-if="!UserModalCtrl.isCreating" class="title">
+      {{ "USERS.UPDATING_USER" | translate }}
+      <label class="label label-warning">{{ "USERS.UPDATE" | translate }}</label>
+    </li>
+  </ol>
+</div>
+
+  <div class="modal-body">
+
+    <div class="form-group" ng-class="{ 'has-error' : UserForm.$submitted && UserForm.display_name.$invalid }">
+      <label class="control-label">{{ "FORM.LABELS.USERNAME" | translate }}</label>
+      <input name="display_name" ng-model="UserModalCtrl.user.display_name" ng-maxlength="UserModalCtrl.maxLength" class="form-control" required>
+      <div class="help-block" ng-messages="UserForm.display_name.$error" ng-show="UserForm.$submitted">
+        <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group"  ng-class="{ 'has-error' : UserForm.$submitted && UserForm.username.$invalid }">
+      <label class="control-label">{{ "FORM.LABELS.LOGIN" | translate }}</label>
+      <input name="username" ng-maxlength="UserModalCtrl.userName" ng-model="UserModalCtrl.user.username" class="form-control" required>
+      <div class="help-block" ng-messages="UserForm.username.$error" ng-show="UserForm.$submitted">
+        <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group" ng-class="{ 'has-error' : UserForm.$submitted && UserForm.email.$invalid }">
+      <label class="control-label">{{ "FORM.LABELS.EMAIL" | translate }}</label>
+      <input name="email" ng-maxlength="UserModalCtrl.length100" ng-model="UserModalCtrl.user.email" class="form-control" type="email" required>
+      <div class="help-block" ng-messages="UserForm.email.$error" ng-show="UserForm.$submitted">
+        <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
+      </div>
+    </div>
+
+    <div class="form-group"
+         ng-class="{ 'has-error' : UserForm.$submitted && UserForm.projects.$invalid }">
+
+      <label class="control-label">{{ "FORM.LABELS.PROJECT" | translate }}</label>
+
+      <ui-select multiple  name="projects" ng-model="UserModalCtrl.user.projects" theme="bootstrap" close-on-select="false" required>
+
+        <ui-select-match placeholder="{{ 'USERS.SELECT_PROJECTS' | translate }}">
+          <span>{{$item.name}}</span>
+        </ui-select-match>
+
+        <ui-select-choices repeat="project.id as project in UserModalCtrl.projects| propsSearch:{name: $select.search}">
```

Comment:
Does Angular's [filter](https://docs.angularjs.org/api/ng/filter/filter) not work in this case?

It seems to me this should work fine:
```html
<ui-select-choices repeat="project.id as project in UserModalCtrl.projects | filter:{name: $select.search}">
```